### ISSUE TITLE
Definition of Done

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#44]
## User story
1. As a team
2. We need to agree on project-specific definition of done (DoD) for feature, sptint and project release
3. So that DoDs can be used for upcoming review and release decisions

## Progress updates
* What i have done so far
* Step 1
* Step 2
* ...

## Acceptance criteria
* Everyone agrees on a DoD
* Inclusion in planning documents

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
